config/routes.rb

```diff
@@ -21,6 +21,8 @@
 
     get '/*route/folder_new', to: 'folders#new', as: :new_folder
     get '/*route/:binary_name', to: 'folders/binaries#show', as: :binary, format: true
+    post '/*route/:binary_name/like', to: 'folders/binaries/likes#create', as: :like_file, format: true
```

Comment:
@case-eee the format: true requires the extension for the file that is uploaded, which is necessary to render it properly on the binary show page, and to make sure it doesn't get mixed up with folder show, because of our use of wildcard segments... i will try a few more combinations to see what breaks when, to see if i can find a way around it